<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddShip</class>
 <widget class="QDialog" name="AddShip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить корабль</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="3" column="1">
      <widget class="QLineEdit" name="Year"/>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Тип судна</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Флаг</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Владелец</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="Height"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="Flag"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Ширина</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Высота</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="Owner"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Год постройки</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="Length"/>
     </item>
     <item row="16" column="0" colspan="2">
      <layout class="QGridLayout" name="gridLayout"/>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="Width"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Длина</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Количество двигателей</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="NameShip"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Название судна</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QListWidget" name="TypeShip">
       <item>
        <property name="text">
         <string>Балкер</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Контейнеровоз</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Танкер</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Сухогруз</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QTableView" name="tableViewEngine"/>
     </item>
     <item row="10" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="AddEngine">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="RemoveEngine">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Обновить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="13" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="Confirm">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Ок</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Cancel">
         <property name="text">
          <string>Отмена</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="12" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Прикрепить файлы</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Confirm</sender>
   <signal>clicked()</signal>
   <receiver>AddShip</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>635</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>653</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Cancel</sender>
   <signal>clicked()</signal>
   <receiver>AddShip</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>634</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>672</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
